package.json

```diff
@@ -0,0 +1,81 @@
+{
+  "name": "ember-brand-manager",
+  "version": "0.0.1",
+  "description": "White-labeling brand manager for Ember.",
+  "keywords": [
+    "ember-addon"
+  ],
+  "repository": "",
+  "license": "MIT",
+  "author": "",
+  "directories": {
+    "doc": "doc",
+    "test": "tests"
+  },
+  "scripts": {
+    "build": "ember build --environment=production",
+    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel \"lint:!(fix)\"",
+    "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*:fix",
+    "lint:hbs": "ember-template-lint .",
+    "lint:hbs:fix": "ember-template-lint . --fix",
+    "lint:js": "eslint . --cache",
+    "lint:js:fix": "eslint . --fix",
+    "start": "ember serve",
+    "test": "npm-run-all lint test:*",
+    "test:ember": "ember test",
+    "test:ember-compatibility": "ember try:each"
+  },
+  "dependencies": {
+    "@embroider/macros": "^1.10.0",
+    "broccoli-funnel": "^3.0.8",
+    "broccoli-merge-trees": "^4.2.0",
+    "ember-cli-babel": "^7.26.8",
+    "ember-cli-htmlbars": "^6.0.1"
+  },
+  "devDependencies": {
```

Comment:
Deps to be removed at this stage:
- broccoli-asset-rev
- ember-cli-sri
- ember-cli-terser
- @glimmer/*
- ember-page-title